assets/component-slider.css

```diff
@@ -24,9 +24,6 @@ slider-component {
     scroll-padding-left: 1rem;
     -webkit-overflow-scrolling: touch;
     margin-bottom: 1rem;
-  }
-
-  .slider--mobile.product__media-list {
     padding-top: calc(var(--shadow-vertical-offset) * -1 + var(--shadow-blur-radius));
     padding-bottom: calc(var(--shadow-vertical-offset) + var(--shadow-blur-radius));
```

Comment:
I think we might always need to have a bit of padding top and bottom whatever the vertical offset value is 🤔  maybe using a `max(0.5rem, calc(var(--shadow-vertical-offset) * -1 + var(--shadow-blur-radius)))` to account for focus styles: 
![](https://screenshot.click/22-01-nl9mn-iydmd.png)

Reply:
Edit: Actually, using max() may straighten that up nicely. Good call. Will update if all looks well 👍

Reply:
Donezo